client/main.go

```diff
@@ -45,6 +46,16 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Verify IP address
+	if utils.VerifyIP(options.Interface.IP) {
```

Comment:
This function should parse input string for IP address validity not `options.Interface.IP`
the `options.Interface.IP` isn't only has IP address. it contain mask.

```go
if utils.VerifyIP(options.Interface.Gateway) 
```